lib/service/database.js

```diff
@@ -66,28 +66,42 @@ var affectedTableRegexCache = {
     pixelHeight: /!pixel_height!/g
 };
 
-DatabaseService.prototype.getLastUpdatedTimeFromAffectedTables = function(node, skip, callback) {
+DatabaseService.prototype.getLastUpdatedTimeAndTableDataFromAffectedTables = function(node, skip, callback) {
```

Comment:
Since we have `CDB_QueryTables_Updated_At` function we can use it to discover affected tables, so reuse it makes sense. But tomorrow, we will need another kind of info from `CDB_QueryTables_Updated_At` (not sure) so we will need to change again the name of this function. What about to rename it from  `getLastUpdatedTimeAndTableDataFromAffectedTables` to `getSourceMetadata`? and return an object as you already implemented, calling it `metadata` (a little less generic) instead of `data`: 

```
metadata = {
  'last_update': 'wadus-date',
  'affected_tables': 'wadus-table'
}
```
I know, it isn't perfect :man_shrugging:, does it make sense?